File: nexed-projecten/php-beginner/04-level/Calculator-upgrade-2d5e9b8f7b41-8e6e2b850de9/calculator.php
<?php

$operator = readline("Welke operatie wil je uitvoeren? (+, -, %): ");
if (!in_array($operator, ['+', '-', '%'])) {
    echo "Fout: geen geldige operatie.\n";
    exit;
}

$getal1 = readline("Eerste getal? ");
if (!is_numeric($getal1)) {
    echo "Fout: geen getal.\n";
    exit;
}

$getal2 = readline("Tweede getal? ");
if (!is_numeric($getal2)) {
    echo "Fout: geen getal.\n";
    exit;
}

$getal1 = floatval($getal1);
$getal2 = floatval($getal2);

if ($operator == "+") {
    $resultaat = $getal1 + $getal2;
    echo "Het resultaat van $getal1 + $getal2 is: $resultaat\n";
} elseif ($operator == "-") {
    $resultaat = $getal1 - $getal2;
    echo "Het resultaat van $getal1 - $getal2 is: $resultaat\n";
} elseif ($operator == "%") {
    if ($getal2 == 0) {
        echo "Fout: Modulo met nul is niet toegestaan.\n";
        exit;
    }
    $resultaat = $getal1 % $getal2;
    echo "Het resultaat van $getal1 % $getal2 is: $resultaat\n";
}

?>

File: nexed-projecten/python-beginner/03-level/Calculator-11-1d96624bfd6e-86b2edf21dd3/calculator.py
operatie = input("Operatie? (+, -, *, /, %): ")
eerstegetal = float(input("Eerste getal? "))
tweedegetal = float(input("Tweede getal? "))

if operatie == "+":
    resultaat = eerstegetal + tweedegetal
elif operatie == "-":
    resultaat = eerstegetal - tweedegetal
elif operatie == "*":
    resultaat = eerstegetal * tweedegetal
elif operatie == "/":
    if tweedegetal == 0:
        resultaat = "Kan niet delen door 0"
    else:
        resultaat = eerstegetal / tweedegetal
elif operatie == "%":
    if tweedegetal == 0:
        resultaat = "Kan niet delen door 0"
    else:
        resultaat = eerstegetal % tweedegetal
else:
    resultaat = f"{operatie} is geen geldige operatie"

print("Resultaat:", resultaat)
